README.md

```diff
@@ -41,16 +47,51 @@ Based on the discussion in [#1](https://github.com/js-reporters/js-reporters/iss
  - `suiteEnd`:  Triggered at the end of each group of tests within a testsuite.
  - `runEnd`:  Indicates the end of a testsuite, triggered just once.
 
-These only define event names, not the data associated with each event. The data still needs to be specified via further discussion in [#1](https://github.com/js-reporters/js-reporters/issues/1).
+These only define event names, not the data associated with each event. The data still needs to be specified via further
+discussion in [#1](https://github.com/js-reporters/js-reporters/issues/1).
 
 #### Selection Criteria
 
 The criteria for picking these event names included:
 
- - These use the most common terms across a selection of frameworks, as gathered in [#1](https://github.com/js-reporters/js-reporters/issues/1#issuecomment-54841874)
- - It uses names that are valid JavaScript identifiers, which allows using those as keys in JSON and avoids the need to quote keys in regular JS objects or function calls.
- - It doesn't overlap with any known existing events, so introducing these could be done in parallel to the existing API in each framework, optionally deprecating and eventually removing the previous API.
-
+ - These use the most common terms across a selection of frameworks, as gathered in
+   [#1](https://github.com/js-reporters/js-reporters/issues/1#issuecomment-54841874)
+ - It uses names that are valid JavaScript identifiers, which allows using those as keys in JSON and avoids the need to
+   quote keys in regular JS objects or function calls.
+ - It doesn't overlap with any known existing events, so introducing these could be done in parallel to the existing API
+   in each framework, optionally deprecating and eventually removing the previous API.
+
+### Event Data
+
+Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there
+are two basic data structures: Suites and Tests. A test represents an atomic test/spec/"`it()`". A suite contains tests
+and optionally other suites. This can represent both flat structures like used by QUnit as well as nested suites like
+used by Jasmine or Mocha. The data structures are defined as follows:
+
+ - **Test**: A test holds basic information on a single test/spec. It has the following set of required attributes
+   - `testName`: name of the test
+   - `suiteName`: name of the suite the test belongs to
+   - `status`: result of the test. Can `passed`, `failed` or `skipped`
```

Comment:
This is redundant with the Suite status below. Let's deduplicate that, either with a separate definition that we references in both places or we define status here and reference it from below.